webapp/portlet/src/main/resources/locale/portlet/Portlets_en.properties

```diff
@@ -698,6 +698,9 @@ profileSettings.label.multiValued=Multi valued
 profileSettings.label.groupSynchronized=Group synchronized
 profileSettings.label.required=Required
 profileSettings.label.active=Active
+profileSettings.label.maskable=Maskable
```

Comment:
I am not sure which will be a better wording ? hiddenable or maskable ?

Reply:
a functional question, in the task they used the label Maskable